src/main/java/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/interfaces/RandomMobDrop.java

```diff
@@ -0,0 +1,27 @@
+package me.mrCookieSlime.Slimefun.Objects.SlimefunItem.interfaces;
+
+import io.github.thebusybiscuit.slimefun4.core.attributes.ItemAttribute;
+
+/**
+ * This interface, when attached to a {@link SlimefunItem}, provides an easy method for adding 
+ * a % chance to drop for an {@link SlimefunItem} on {@link entityDeathEvent}, this chance is 0-100
+ * and used in conjunction with the MOB_DROP {@link RecipeType}. 
+ * @see BasicCircuitBoard and @see MobDropListener.

```

Comment:
No "and" here. The syntax is as follow:

@ see x
@ see y

And we usually place it below the @ author tag.